core/sqf/conf/log4cxx.trafodion.masterexe.config

```diff
@@ -44,15 +45,15 @@ log4j.appender.mxoAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.mxoAppender.layout.ConversionPattern=%d, %p, %c, %m%n
 
 #MXOSRVR
+log4j.additive.mxoAppender=false 
 log4j.logger.MXOSRVR=ERROR, mxoAppender
 
 # SQL
 # uncomment the following line if you want to enable debug logging
-#log4j.logger.SQL.EXE=DEBUG
 log4j.logger.SQL=INFO
 log4j.logger.SQL.COMP=ERROR
-log4j.logger.SQL.EXE.Memory=INFO
-log4j.logger.SQL.ESP=ERROR
+log4j.logger.SQL.EXE=INFO
+log4j.logger.SQL.ESP=INFO
```

Comment:
Ok . But we are definitely changing behaviour already. There is a generic method SQLMXLoggingArea::logExecRTInfo that uses INFO as the level. Am guessing it could affect performance since it's used in BMO code paths.    Not sure we wnt the user to see all these messages by default. If you do a  quick grep on our code you can see what is being logged using this generic method.